reference/changelog.md

```diff
@@ -87,6 +92,18 @@ redirect_from: /install/changelog.html
     </tbody>
 </table>
 
+## v0.16.3 {#v163}
+
+Released on November 6, 2018
+
+### Pulumi CLI
+
+There are no changes from the previous version of the Pulumi CLI, but it shares the same version as the `@pulumi/pulumi` package, which was updated
+
+### @pulumi/pulumi 0.16.3
+
+- Support closure seralization when running on Node 11 [pulumi/pulumi#2101](https://github.com/pulumi/pulumi/pull/2101)
```

Comment:
How about we just say - "Fully support Node.js 11".  (folks may not realize that closure serialization was the only thing not yet working on Node 11).

Reply:
Can do!